src/stress/memory/oom.spec.ts

```diff
@@ -0,0 +1,68 @@
+export const description = `
+Stress tests covering robustness when available VRAM is exhausted.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+// Helper to exhaust VRAM until there is less than 64 MB of capacity. Returns
+// an opaque closure which can be called to free the allocated resources later.
```

Comment:
nit: Use `/** tsdoc-style comments */` so this will be picked up by editors (and documentation generators, if this helper were in a different file).